build/android/BUILD.gn

```diff
@@ -30,6 +30,19 @@ action("sign_app") {
   ]
 }
 
+action("collect_objs_app") {
+  script = "//brave/build/android/collect_obj_files.py"
+
+  deps = []
+  target_obj_files_name = "//out/Default$target_cpu$target_apk_base"
+
+  outputs = [ "$root_out_dir/$target_obj_files_name" ]
```

Comment:
I think there's a problem here with missing inputs to trigger gn to run the target. @SergeyZhukovsky says it works correctly, but I'm not sure how that is possible so testing now

Reply:
made a correct output target